versions.yaml

```diff
@@ -88,8 +88,8 @@ assets:
     qemu:
       description: "VMM that uses KVM"
       url: "https://github.com/qemu/qemu"
-      version: "v5.2.0"
-      tag: "v5.2.0"
+      version: "v6.1.0-rc3"
```

Comment:
snap CI is failing because of this change

```
INFO: Apply patches from /home/runner/work/kata-containers/kata-containers/stage/gopath/src/github.com/kata-containers/kata-containers/tools/packaging/qemu/patches/6.1.0-r.x
INFO: Patches directory does not exist: /home/runner/work/kata-containers/kata-containers/stage/gopath/src/github.com/kata-containers/kata-containers/tools/packaging/qemu/patches/6.1.0-r.x
INFO: Create a /home/runner/work/kata-containers/kata-containers/stage/gopath/src/github.com/kata-containers/kata-containers/tools/packaging/qemu/patches/6.1.0-r.x/no_patches file if the current version has no patches
```

Reply:
Well, yes, but I'm not clear why.  I'm not sure why the snap build is using different logic than the rest of the build_qemu logic which is working with the change.